ln1_f4.py

```diff
@@ -1,115 +1,108 @@
-#!/usr/bin/python
-#coding: utf8
-#Програма повинна запускатися кожних 5хв (12 запусків за год) і записувати статистику використання ресурсів в БД, через годину 
-#дані з БД усереднюються та відправляються на електронку у вигляді таблиці з статистикою
+# !/usr/bin/python
+# coding: utf8
 
+import os
+import json
+import datetime
 import smtplib
 import subprocess
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
-data_string = [] #Усі дані зберігаються у форматі cpu_total,cpu_user,cpu_sys,cpu_idle,mem_total,mem_used,mem_free,mem_cached,hdd_total,hdd_used,hdd_free
 
-def get_raw_data(command): #функція виконує консольну команду та повертає результат
+def get_sysresinfo(command):  # функція виконує консольну команду та повертає результат
     command_data = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
     (command_data, err) = command_data.communicate()
-    command_data = command_data.replace(",",".") #Заміна ком на крапки для коректного виконання float()
+    command_data = command_data.replace(",", ".")  # Заміна ком на крапки для коректного виконання float()
     return command_data.split()
 
-raw = get_raw_data("mpstat") #обробка даних по завантаженості процесора
+raw = get_sysresinfo("mpstat")  # обробка даних по завантаженості процесора
```

Comment:
line 19-26 - group this into function (`get_cpu_load`). After that you don't need comment.